static/js/buddy_data.js

```diff
@@ -227,11 +215,22 @@ exports.info_for = function (user_id) {
         is_current_user: people.is_my_user_id(user_id),
         num_unread: get_num_unread(user_id),
         user_circle_class: user_circle_class,
-        title: title,
         user_circle_status: user_circle_status,
     };
 };
 
+exports.register_title_tooltip = function (elem, user_id) {
```

Comment:
Why do we call this `register_title_tooltip` instead of `show_buddy_list_title`?  I don't understand where the "register" part comes from.

Reply:
I'll make this change ASAP. 
Tim suggested the [name](https://chat.zulip.org/#narrow/stream/91-code-review/topic/.2311882/near/717306) for the function in one of the early reviews. It was for a more generic function though, and I didn't think about changing this when I modified this function to only generate specific tooltips. 
Thanks for pointing it out.